zerver/models.py

```diff
@@ -751,6 +759,17 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
         wildcard_mention_policy=int,
     )
 
+    REALM_PERMISSION_GROUP_SETTINGS: Dict[str, GroupPermissionSetting] = dict(
+        create_multiuse_invite_group=GroupPermissionSetting(
+            require_system_group=True,
+            allow_internet_group=False,
+            allow_owners_group=False,
```

Comment:
@sahil839 is there a reason `allow_owners_group` should be false here? I can see situations where one might want to limit it to just owners, but wasn't sure if there was some policy situation to think about here.

Reply:
I am not sure. I thought it was described as such in the issue, but it isn't. I don't think there would be any issues to allow `Owners only` option. @Ujjawal3 can you confirm if we are missing something here? If not then we can set `allow_owners_group` as true.